ui/src/components/imquery/IMQueryEditor.vue

```diff
@@ -15,34 +15,18 @@
       <div class="feature-title side-title">Features:</div>
       <div class="feature-list">
         <div class="feature-list-container">
-          <div class="feature">
-            <MatchDisplay
-              class="feature-description clickable"
-              :match="editQueryDefinition"
-              :class="[hover ? 'feature-description-card-hover' : 'feature-description-card']"
-              @mouseover="mouseover"
-              @mouseout="mouseout"
-              @click="editMatch"
-            />
-          </div>
-        </div>
-        <div class="add-buttons">
-          <Button label="Add parent cohort" @click="showAddPopulation = true" severity="help" icon="fa-solid fa-user-group" class="add-feature-button" />
-          <Button
-            label="Add existing feature"
-            @click="showAddFeature = true"
-            severity="success"
-            icon="fa-solid fa-plus"
-            class="add-feature-button"
-            v-tooltip.bottom="'Add definition from existing feature'"
+          <EditMatch
+            :edit-match="editQueryDefinition"
+            :is-boolean-editor="true"
+            :is-root-feature="true"
+            class="feature-description clickable"
+            @on-update-dialog-focus="menuItems => editMatch(menuItems)"
```

Comment:
shouldn't need to capture input.  ="editMatch" should be enough to run with inputs